src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -98,7 +98,7 @@ public override void OnStartActivity(Activity activity, object payload)
                     // Starting the new activity make it the Activity.Current one.
                     newOne.Start();
 
-                    newOne.SetTag("IsCreatedByInstrumentation", true);
+                    newOne.SetTag("IsCreatedByInstrumentation", "True");
```

Comment:
i think u could replace for bool.True (something like this)

Reply:
Fixed this.